Sources/SpelledPitch/PitchSpeller/Wetherfield/Wetherfield.swift

```diff
@@ -32,13 +32,37 @@ struct PitchSpeller {
             UnorderedPair(.down,.down)
         ]
     )
-
-    static let adjacencyCarrying = AdjacencyCarrying.build(from: PitchSpeller.tendencyGraph)
-    static let tendencyMask: AdjacencyCarrying<Graph<FlowNode<Cross<Int,Tendency>>>> = adjacencyCarrying.pullback { node in
+    
+    static let eightTendencyLink: [(Pitch.Class, Tendency)] = [
+        (.init(value: 0), .down),
```

Comment:
Because `Pitch.Class` conforms to `ExpressibleByIntegerLiteral`, you can actually just use:

```Swift
(0, .down), ...
```

Reply:
Sweet!